TeamCode/src/main/java/org/firstinspires/ftc/teamcode/tutorials/TissueBoxExample.java

```diff
@@ -0,0 +1,29 @@
+package org.firstinspires.ftc.teamcode.tutorials;
+
+public class TissueBoxExample {
+    int MAX_TISSUES = 100;
+    int currentTissueCount;
+
+    public void init() {
```

Comment:
Wow! I love public void inits!

Reply:
Hurray!

Reply:
I'm going to need a tissue because this code is so good 🥲